microsoft/nni
# Refactor log components(remove log collection)

### Description ###
- main content: PR title
- other content: 
  - use function component to rewrite trial log components
  - delete some unused className, css


### How to test ###
run experiments to see trial log. please run experiments that `training services === local` and `training services !== local`
